staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5891,6 +5893,15 @@ type NodeRuntimeHandler struct {
 	Features *NodeRuntimeHandlerFeatures `json:"features,omitempty" protobuf:"bytes,2,opt,name=features"`
 }
 
+// NodeFeatures describes the set of features implemented by the CRI implementation.
+// The features contained in the NodeFeatures should depend on only cri implementation, be independent of runtime handlers,
+// (i.e. it should not require inspecting any information from the OCI runtime-spec's features).
```

Comment:
Same comment as above.

Reply:
Fixed in 
- K8s API: 5d75660dc11ff443ebab2551aed8e56a54cc218d
- CRI: 38e4c6b01666128216290e1579a8f2d408a60e66